migrations/v0.7/README.md

```diff
@@ -2,7 +2,7 @@
 
 > NOTE: This applies when upgrading to the latest `v0.7.x` release.
 
-When migrating from Vela version [v0.6.0](../../releases/v0.6.0.md) to [v0.7.5](../../releases/v0.7.5.md) the Vela administrator will want to ensure the following actions are being performed:
+When migrating from Vela version [v0.6.0](../../releases/v0.6.0.md) to [v0.7](../../releases/v0.7.md) the Vela administrator will want to ensure the following actions are being performed:
```

Comment:
Is there a reason to not be consistent when using patch versions?  Can this just be `v0.6` as seems to be what the other migration doco's use

Reply:
Though now that i say something I notice this is `migrations/v0.7` vs the one's I was referencing `releases/v0.7.md` so maybe my point doesn't make any sense :)

Reply:
Yes, there is a reason not to be consistent right now.. but it is a temporary one 👍 

At the time of this PR, the `releases/v0.6.md` document has not been created yet 😞 

You can verify this here:

https://github.com/go-vela/community/tree/master/releases

I was trying to keep the PRs smaller by doing them in batches per release.

So I've opened up a PR to condense all `v0.8` release notes:

https://github.com/go-vela/community/pull/290

This one is condensing all `v0.7` release notes.

Then a subsequent PR will be opened to condense all `v0.6` release notes.

Once the PR to condense all `v0.6` release notes is submitted, I will modify this line to match the others 👍

Reply:
:amaze: 👍